### PR TITLE
Fix flexbox on celltoolbar and safari.

### PR DESCRIPTION
**Pull request by Carreau**
cf https://github.com/jupyter/nbgrader/issues/394

Cannot fix in flexbox.less cf above issue.

Ping @jhamrick

### CODE REVIEW

**Comment by jhamrick** [comment]
Thanks! I'll have a look at this in just a bit.

**Comment by jhamrick** [comment]
Can you let me know what the right fix for the nbgrader css is in conjunction with this? There's no change without it, so I can't really test this out otherwise.

**Comment by Carreau** [comment]
> Can you let me know what the right fix for the nbgrader css is in conjunction with this? There's no change without it, so I can't really test this out otherwise.

This shouldn't need any change on your side. 

I'll try to issue a PR against nbgrader that also fix it.

File: notebook/static/notebook/less/celltoolbar.less [hunk 1]
```diff
@@ -12,6 +12,10 @@
     padding-right: 4px;
     .hbox();
     .end();
+    // safari fix, we cannot use -webkit-flex on hbox
+    // and vbox either or all css get borked
+    // cf https://github.com/jupyter/nbgrader/issues/394
+    display: -webkit-flex;
```
Comment (minrk) [inline notebook/static/notebook/less/celltoolbar.less]: shouldn't this be in `.hbox()` instead, if it's needed?

Comment (Carreau) [inline notebook/static/notebook/less/celltoolbar.less]: Technically that what we should do, but as I stated in the
comment it is broken on safari if in hbox or vbox. 
Reposted screenshot on linked issue:

Comment (minrk) [inline notebook/static/notebook/less/celltoolbar.less]: Sorry, I didn't understand the comment. Okay, then. *grumbles about CSS*

Comment (Carreau) [inline notebook/static/notebook/less/celltoolbar.less]: > grumbles about CSS

indeed.

**Pull request merged**